src/client/websocket/WebSocketShard.js

```diff
@@ -381,10 +373,9 @@ class WebSocketShard extends EventEmitter {
          */
         this.emit(ShardEvents.RESUMED);
 
-        this.trace = packet.d._trace;
         this.status = Status.READY;
         const replayed = packet.s - this.closeSequence;
-        this.debug(`RESUMED ${this.trace.join(' -> ')} | Session ${this.sessionID} | Replayed ${replayed} events.`);
+        this.debug(`RESUMED Session ${this.sessionID} | Replayed ${replayed} events.`);
```

Comment:
```suggestion
        this.debug(`RESUMED | Session ${this.sessionID} | Replayed ${replayed} events.`);
```

Reply:
Sorry for the nitpicking!

Reply:
Nitpicking is appreciated, I had no clue how I wanted these to look.